src/schema/provenance_schema.yaml

```diff
@@ -645,7 +645,9 @@ ENTITIES:
         description: "The displayname of globus group which the user who created this entity is a member of"
         before_create_trigger: set_group_name #same as group_uuid, except set group_name
       previous_revision_uuid:
-        type: string
+        type:
+          - string
+          - list
```

Comment:
You only added this multi data types to `Publication`, will need it for `Dataset.previous_revision_uuid` too.